packages/flutter_tools/lib/src/base/common.dart

```diff
@@ -7,16 +7,14 @@ import 'platform.dart';
 
 /// Return the absolute path of the user's home directory
 String get homeDirPath {
-  if (_homeDirPath == null) {
-    _homeDirPath = platform.isWindows
-        ? platform.environment['USERPROFILE']
-        : platform.environment['HOME'];
-    if (_homeDirPath != null)
-      _homeDirPath = fs.path.absolute(_homeDirPath);
+  String _homeDirPath = platform.isWindows
```

Comment:
Since it's a local variable now, there's no need for the leading underscore, and you can simplify its name, so maybe just `path`?